src/main/resources/com/google/api/codegen/nodejs/main.snip

```diff
@@ -58,10 +58,12 @@
     constructor(opts) {
       this._descriptors = {};
 
-      const servicePath =
-        opts.servicePath ||
-        opts.apiEndpoint ||
-        this.constructor.servicePath;
+      let servicePath = this.constructor.servicePath;
```

Comment:
I'll sometimes start with:

```js
opts = opts || {}
```

to eliminate boiler plate, might be worth considering; definitely a nit, if you had a good reason for the if/else.

Reply:
No good reason! Changing to what you suggested.